kong/plugins/aws-lambda/handler.lua

```diff
@@ -51,6 +52,17 @@ function AWSLambdaHandler:access(conf)
     query = conf.qualifier and "Qualifier=" .. conf.qualifier
   }
 
+  if conf.aws_iam_role then
```

Comment:
The behavior in the upstream plugin is wrong. This should be a breaking change in particular of you target the 11 release